From: Dmitry Bogatov <KAction@debian.org>
Date: Tue, 12 Mar 2019 19:07:02 +0000
Subject: Emulalate sysv runlevel 5 by default

Forwarded: not-needed
Last-Update: 2016-06-09

Currently, not many daemons in Debian provides proper runscripts, so
if user switches to runit, she will on next reboot, that none of daemons,
including essential, like cron, are started.

This patch starts them in sysv-init style -- unmanaged, in background.
It is sub-par solution, but allows one-by-one transition.
Last-Update: 2016-06-09
---
 runit-2.1.2/etc/debian/1 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/etc/debian/1 b/etc/debian/1
index fd88924..7f71b30 100755
--- a/etc/debian/1
+++ b/etc/debian/1
@@ -8,3 +8,5 @@ PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
 touch /run/runit.stopit
 chmod 0 /run/runit.stopit
+
+[ -e /etc/runit/no-emulate-sysv ] || /etc/init.d/rc 5
